qiskit_nature/problems/sampling/protein_folding/interactions/miyazawa_jernigan_interaction.py

```diff
@@ -0,0 +1,47 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""A class defining a Miyazawa-Jernigan interaction between beads of a peptide."""
```

Comment:
Do we need to cite some paper here?

Reply:
We already cite the corresponding paper in the interaction table txt file. I am not sure if needed here!

Reply:
That's a good idea, thanks.